src/BuildScriptGenerator/DefaultBuildScriptGenerator.cs

```diff
@@ -156,6 +156,17 @@ internal class DefaultBuildScriptGenerator : IBuildScriptGenerator
                     }
 
                     platformVersion = detectionResult.LanguageVersion;
+                    _logger.LogDebug(
+                        "Using user supplied platform {platformName} and detected version {platformVersion}.",
+                        userSuppliedPlatform,
```

Comment:
will we log those as dimensions as well? It makes it easier to write queries IMO

Reply:
yes, all those names in `{` and `}` get logged in dimensions